nucypher/cli/commands/stake.py

```diff
@@ -1349,3 +1351,48 @@ def mint(general_config: GroupGeneralConfig,
                           emitter=emitter,
                           chain_name=blockchain.client.chain_name,
                           transaction_type='mint')
+
+
+@stake.command()
+@group_transacting_staker_options
+@option_config_file
+@option_force
+@group_general_config
+@click.option('--external-staker-address',
+              help="The staking address of a staker to manually migrate", type=EIP55_CHECKSUM_ADDRESS)
+def migrate(general_config: GroupGeneralConfig,
```

Comment:
We should add a docstring so that it is used as the help text for the CLI

Reply:
✔️